eventuate-tram-examples-jdbc-kafka/build.gradle

```diff
@@ -0,0 +1,50 @@
+plugins {
+    id "io.spring.dependency-management" version "1.0.6.RELEASE"
+}
+
+repositories {
+    mavenCentral()
+    maven { url "https://jcenter.bintray.com" }
+}
+
+dependencyManagement {
+    imports {
+        mavenBom 'io.micronaut:micronaut-bom:1.1.0'
+    }
+}
+
+dependencies {
+    testCompile project(":eventuate-tram-examples-common")
+
+    testCompile "io.eventuate.messaging.kafka:eventuate-messaging-kafka-consumer-micronaut:$eventuateMessagingKafkaVersion"
+    testCompile "io.eventuate.messaging.kafka:eventuate-messaging-kafka-common-micronaut:$eventuateMessagingKafkaVersion"
+    testCompile "io.eventuate.messaging.kafka:eventuate-messaging-kafka-basic-consumer-micronaut:$eventuateMessagingKafkaVersion"
```

Comment:
These dependencies seem very numerous and complicated.

On the consumer side there is all this:

Why depend on `eventuate-messaging-kafka-consumer-micronaut` and `eventuate-messaging-kafka-basic-consumer-micronaut`. I would have expected `eventuate-messaging-kafka-consumer-micronaut` to already depend on `eventuate-messaging-kafka-basic-consumer-micronaut`.

I would also have expected `eventuate-messaging-kafka-consumer-micronaut`  to depend on `eventuate-messaging-kafka-common-micronaut`

And, then I see this `eventuate-tram-consumer-kafka-micronaut` - surely this depends on all of the above.

And on the producer side there is:

    testCompile "io.eventuate.tram.core:eventuate-tram-jdbc-micronaut:$eventuateTramVersion"
    testCompile "io.eventuate.tram.core:eventuate-tram-producer-jdbc-micronaut:$eventuateTramVersion"

`eventuate-tram-producer-jdbc-micronaut` already depends upon `eventuate-tram-jdbc-micronaut`

Reply:
I will fix